### ISSUE TITLE
Clarify ambiguity regarding when VRFrameData is refreshed

### ISSUE THREAD

**Issue by kearwood** [w3c/webvr#112]
Could we clarify some ambiguity regarding when VRFrameData is refreshed?
- Does calling VRDisplay.ResetPose result in VRFrameData being refreshed mid-frame, or on the next frame?
- Before starting VR presentation, should GetFrameData always return latest data?
  Should GetFrameData() return updated values when called immediately after a - VRDisplay.SubmitFrame call within a single VRDisplay RAF callback?

**Comment by RafaelCintron** [comment]
@kearwood , for your second bullet point, my preference would be to have getFrameData only work (return true and fill in the object) in the context of a RAF callback and only before calling submitFrame. Outside of RAF and after submitFrame, you're not in a frame so filling in "frame data" makes no sense.

**Comment by kearwood** [comment]
In the 2017-01-11 WebVR Implementers F2F, we had a breakout session to discuss this.

Conclusions:
- Agree that outside of the RAF callback, GetFrameData should return false and not update the passed 
- Agree that GetFrameData will return the same values until the next RAF callback, regardless of if submitFrame is called
- "Magic Window" experiences that wish to use pose information can have a RAF callback without calling submitFrame and still get a new set of data on each RAF callback

**Comment by toji** [comment]
Closing since this is planned to be addressed in 2.0. If anyone has issues with the 2.0 proposal, please file a new issue. For details see: 
https://github.com/w3c/webvr/blob/master/explainer.md#main-render-loop
